src/resources/elements/primeDesignSystem/pform-input/pform-input.scss

```diff
@@ -0,0 +1,55 @@
+@import "../styles";
+
+pform-input {
+  .formInputLabel {
+    display: flex;
+    color: $Secondary02;
+    margin-bottom: $spacing-normal;
+    align-items: center;
+    justify-content: space-between;
+
+    .formInputLabelTitle {
+      font-size: $font-size3;
+      font-weight: 700;
+    }
+    .formInputLabelExtra {
+      color: $Neutral02;
+      font-size: $font-size2;
+    }
+  }
+
+  .labelDescription {
+    color: $Neutral02;
+  }
+
+  .inputContainer {
+    margin-top: 10px;
+  }
+
+  .helperMessage {
+    color: $Neutral02;
+    font-size: $font-size2;
+  }
+
+  .feedback-message {
```

Comment:
Might need to align on naming `camelCase` vs. `kebab-case`

Reply:
Yes. I am used with `kebab-case` and Doug told me you are using `camelCase` in the project. If any `kebab-case` then it's my fault as my muscle memory just writes that automatically. I will change it. Please point to any of these things.

Also, maybe we can add a linter for SCSS as well.